eng/Version.Details.xml

```diff
@@ -1,16 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
+    <Dependency Name="System.Text.Json" Version="8.0.0-alpha.1.22631.1">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>a6f84d42529494b3539684ed993fa554281215b7</Sha>
+      <SourceBuild RepoName="runtime" ManagedOnly="false" />
+    </Dependency>
+    <!-- MSBuild versions starting from 16.10.0 do not target NET Core -->
+    <Dependency Name="Microsoft.Build" Version="16.8.0">
+      <Uri>https://github.com/dotnet/msbuild</Uri>
+      <Sha>a6f6699d1f70bf79db82030938d2c5e52d1e4d2e</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.Build.Tasks.Core" Version="16.8.0">
+      <Uri>https://github.com/dotnet/msbuild</Uri>
+      <Sha>a6f6699d1f70bf79db82030938d2c5e52d1e4d2e</Sha>
+    </Dependency>
   </ProductDependencies>
   <ToolsetDependencies>
     <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="8.0.0-beta.22630.1">
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>e82404fca08383513e0b0b3c5308d4a9b18b7c7a</Sha>
+      <SourceBuild RepoName="arcade" ManagedOnly="true" />
     </Dependency>
     <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="8.0.0-alpha.1.23055.1">
```

Comment:
Since you are touching this file, can you make this a ProductDependency?

Reply:
Yeah, thanks for the reminder, will change